### DISCUSSION TITLE
It is related to the concept list in the Start training section in dreambooth colab

### DISCUSSION THREAD

**Discussion by uncleofcadi** [TheLastBen/fast-stable-diffusion#1815] [Q&A]
Hello. I want to produce 2D assets for a game and I have 33 different kinds of cupcake images. I would like to introduce these cupcakes separately in Colab.
For this, I thought of extending the concept list, but I have no idea how to do it. I will be glad if you help me.

**Comment by TheLastBen** [comment]
if you want to replicate the exact same cupcakes, you will need to train each cupcake with its own token so you can call it easily at inference time, just like you would train a face.

**Reply by uncleofcadi** [reply]
![Screenshot 2023-03-23 at 14 41 44](https://user-images.githubusercontent.com/115935273/227193456-d0970a27-4b83-4c8f-a6f5-3256bbcbf6e0.png)


First of all, I would like to thank you very much for your answer, for example, I want to put a banana on a cupcake, strawberry on another is something else, but I want the cupcake art styles to remain the same. Should I edit the part in the photo separately for each cupcake as it appears? Should they all have different names?

**Reply by TheLastBen** [reply]
this is not my notebook, you can get mine from the main page of this repo by clicking on the thumbnail of dreambooth
